app/utils/staticData/grupos.js

```diff
@@ -255,7 +255,7 @@ export default [
   },
   {
     id: 'tecnopolis2021ConCoty',
-    desafioIds: [2021201, 2021202, 2021203, 2021204, 2021205, 2021206, 2021207, 255]
+    desafioIds: [2021201, 2021202, 2021203, 2021204, 2021205, 2021206, 2021207]
   },
   {
```

Comment:
Aca saque el 255, el de coty dibuja libre, de los desafios de tecnolopolis porque estaba repetido en otro grupo. Lo que pasaba es que este desafio solo se mostraba en este grupo de tecnopolis y no el ultimo grupo del libro 1. 
Lo que pasa es que en el model de desafio se define una relacion many to one, donde los desafios pertenecen a un solo grupo `belongsTo('grupo')`. Entonces esto tiene sentido que esté pasando, un desafio solo puede ser de un grupo, lo que creo que no tiene sentido es por que es que andaba antes.